src/Controller/PetController.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Pet;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Annotation\Route;
+use App\Form\PetForm;
+
+class PetController extends AbstractController
+{
+    /**
+     * @Route("/new", name="new_pet")
+     * @param Request $request
+     * @return \Symfony\Component\HttpFoundation\RedirectResponse|\Symfony\Component\HttpFoundation\Response
+     */
+    public function new(Request $request)
+    {
+        // creates a pet and creates a form for adding new pet
+        $pet = new Pet();
+        $form = $this->createForm(PetForm::class, $pet);
```

Comment:
```suggestion
        $form = $this->createForm(PetForm::class, $pet = new Pet());
```

I would do this instead but that is my personal perference.